website/source/docs/internals/scheduling/scheduling.html.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: "docs"
 page_title: "Scheduling"
-sidebar_current: "docs-internals-scheduling"
+sidebar_current: "docs-internals-scheduling-internals"
```

Comment:
Delete the copy that is now in the index.html?

Reply:
index.html is only a small snippet of content that links to the scheduling and preemption pages.